WordPress/twentytwentyfive
# Fix: Improve the mobile version of "Events - List with Call to Action" pattern.

**Description**

Partial to https://github.com/WordPress/twentytwentyfive/issues/302

Improve the mobile version of "Events - List with Call to Action" pattern by allowing it to wrap into multiple lines.


**Screenshots**



https://github.com/user-attachments/assets/faa04e2b-df39-4517-97a3-ce94906cbdde

**Testing Instructions**

1. Create a page.
2. Insert the "Events - List with Call to Action" pattern.
3. Save the page and view it on mobile viewport.
4. Confirm that the section of the date and button is wrapping into multiple lines, and not narrowing down together.
